<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charts</class>
 <widget class="QMainWindow" name="charts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GraphCreator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="txtTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>559</width>
      <height>115</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ChartCrafter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblBarChart">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <row>
     <property name="text">
      <string>Bar 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bar 10</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnBarChart">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBarChartTitle">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPieChart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPieChartTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblPieChart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <row>
     <property name="text">
      <string>Slice 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Slice 10</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Label</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="txtBarChartLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bar Chart</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtPieChartLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pie Chart</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLineChart">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>580</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLineChart">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Line Chart</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLineChartTitle">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblLineChart">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>350</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <row>
     <property name="text">
      <string>Point 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Point 20</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X Val</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y Val</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="txtBarChartLabelX">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>X-Axis Label</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBarChartLabelY">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Y-Axis Label</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLineChartLabelX">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>250</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>X-Axis Label</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLineChartLabelY">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Thin</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Thin'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Y-Axis Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
